kubescape.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: kubescape
+  version: 2.0.182
```

Comment:
```suggestion
  version: 2.0.183
```

Reply:
thanks @dwertent, I've upgraded it accordingly.